editor/plugins/sprite_editor_plugin.cpp

```diff
@@ -91,6 +91,8 @@ Vector<Vector2> expand(const Vector<Vector2> &points, const Rect2i &rect, float
 
 	Vector<Vector2> outPoints;
 	ClipperLib::PolyNode *p2 = out.GetFirst();
+	if (p2 == NULL) // avoid editor crash
```

Comment:
Above for `p2` there is:
```
ERR_FAIL_COND_V(!p, points);
```
So I guess it might be better to return `points` when aborting? Don't know if it should be made an `ERR_FAIL` on the other hand. If yes, that means there's still a bug to fix (what makes it fail).

Reply:
Well given your OP comment, I guess it should be an `ERR_FAIL_COND_V` indeed, since it only prevents the crash, doesn't fix the actual bug.

Reply:
Hmm. however its possible to actually use Sprite with Region but not through this sub-editor

Reply:
Yeah, fixed it as you said